h2o-dist/buildinfo.json

```diff
@@ -20,91 +20,109 @@
     "hadoop_distributions" : [
         {
             "distribution" : "cdh5.4",
+            "hiveVersion"  : "hive", 
```

Comment:
I think this should be elsewhere, the information is relevant only to our images. Probably the best place would be https://github.com/h2oai/h2o-3/blob/fef47a9386b6971190e2967a6d3f8cdb5dd86b2d/scripts/jenkins/groovy/pipelineUtils.groovy#L84.

Reply:
removed